Cargo.toml

```diff
@@ -25,7 +25,7 @@ debug = ['serde_json']
 [dependencies]
 anyhow = "1.0"
 bincode = "1.3.1"
-clap = { version = "2.33.3", default_features = false, features = ["vec_map"] }
```

Comment:
From https://github.com/clap-rs/clap/releases/tag/v3.0.0 I think this should still disable default_features. I don't mind adding deps if we need them but there's ~15 new dependencies in Cargo.lock. It's also increased the release binary size by ~100kb.

Reply:
The features we can disable now are `color` for coloring the help output and `suggestions` for suggesting "off by a letter" flag typos etc., let me know if you'd like to keep any of these

Reply:
Can you do https://github.com/dimo414/bkt/blob/02dbd9c5b7510c2091463c5d75e406c18e792cbf/Cargo.toml#L28 ? I'd like to opt-out of the default features except those we need.